qiskit/quantum_info/operators/operator.py

```diff
@@ -141,6 +141,71 @@ def settings(self):
             "output_dims": self.output_dims(),
         }
 
+    def draw(self, output=None, **drawer_args):
```

Comment:
Hi @ikkoham, thanks for your feedback. The reason I added these lines of code is that, another feature that we need to add is the ability to allow the user to change the drawer options thru the user config file. The final version of the code will resemble what's being done for statevectors [here](https://github.com/Qiskit/qiskit-terra/blob/7df290b8f3245510cae2393c999dd8ee8a4037dc/qiskit/visualization/state_visualization.py#L1476-L1482), but after talking to Jake Lishman, that feature will be added thru a separate PR because it requires change of code in other files, separate tests, etc.

I can definitely remove the code, and re-add it later in that other PR. Let me know your thoughts.

Reply:
I understood. Then, default argument `None` seems to be nice. Thank you.